src/app/tests/suites/certification/Test_TC_LVL_2_1.yaml

```diff
@@ -23,116 +23,140 @@ tests:
       cluster: "DelayCommands"
       command: "WaitForCommissionee"
 
-    - label: "reads current Level attribute from DUT"
+    #Disabled due to issue - #11670
+    - label: "Reads the CurrentLevel attribute"
+      disabled: true
       command: "readAttribute"
-      attribute: "current Level"
+      attribute: "current level"
       response:
-          value: 254
+          value: null
 
-    - label: "sends a Move to level command"
-      command: "MoveToLevel"
-      arguments:
-          values:
-              - name: "level"
-                value: 64
-              - name: "transitionTime"
-                value: 0
-              - name: "optionMask"
-                value: 1
-              - name: "optionOverride"
-                value: 1
+    - label: "Reads the CurrentLevel attribute"
+      command: "readAttribute"
+      attribute: "current level"
+      response:
+          constraints:
+              type: uint8
 
-    - label: "Wait 100ms"
-      cluster: "DelayCommands"
-      command: "WaitForMs"
-      arguments:
-          values:
-              - name: "ms"
-                value: 100
-
-    - label: "reads current Level attribute from DUT"
-      command: "readAttribute"
-      attribute: "current Level"
-      response:
-          value: 64
-
-    - label: "sends a Move to level command"
-      command: "MoveToLevel"
-      arguments:
-          values:
-              - name: "level"
-                value: 128
-              - name: "transitionTime"
-                value: 1
-              - name: "optionMask"
-                value: 1
-              - name: "optionOverride"
-                value: 1
-
-    - label: "Wait 200ms"
-      cluster: "DelayCommands"
-      command: "WaitForMs"
-      arguments:
-          values:
-              - name: "ms"
-                value: 200
+    - label: "Reads the RemainingTime attribute"
+      command: "readAttribute"
+      attribute: "remaining time"
+      response:
+          value: 0
+          constraints:
```

Comment:
Right now, if both "value" and "constraints" is set the "constraints" gets ignored, afaict.  If you look for the generated code for this function, it just checks the value, not the type.

Is that expected?  If not, is there an issue tracking checking both "value" _and_ "constraints"?  Or should this be two separate tests, one checking the value one checking the constraints?   Because as things stand, this test is not testing what it thinks it's testing.

@vivien-apple

Reply:
we will file an issue for that